Source/WebKit/PlatformWin.cmake

```diff
@@ -4,6 +4,8 @@ set(NetworkProcess_OUTPUT_NAME WebKitNetworkProcess)
 set(GPUProcess_OUTPUT_NAME WebKitGPUProcess)
 set(PluginProcess_OUTPUT_NAME WebKitPluginProcess)
 
+set(WebKit_USE_PREFIX_HEADER ON)
```

Comment:
WinCairo EWS failed.

> FAILED: bin64/WebKitWebProcess.exe 
> cmd.exe /C "cd . && C:\tools\cmake\bin\cmake.exe -E vs_link_exe --intdir=Source\WebKit\CMakeFiles\WebProcess.dir --rc="C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64\rc.exe" --mt="C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64\mt.exe" --manifests  -- C:\MSVS\VC\Tools\MSVC\14.33.31629\bin\Hostx64\x64\link.exe /nologo Source\WebKit\CMakeFiles\WebProcess.dir\WebProcess\EntryPoint\win\WebProcessMain.cpp.obj  /out:bin64\WebKitWebProcess.exe /implib:lib64\WebKitWebProcess.lib /pdb:bin64\WebKitWebProcess.pdb /version:0.0 /machine:x64 /DEBUG /OPT:ICF /OPT:REF /INCREMENTAL:NO /INCREMENTAL:NO /subsystem:console  "/manifestdependency:type='win32' name='Microsoft.Windows.Common-Controls' version='6.0.0.0' processorArchitecture='amd64' publicKeyToken='6595b64144ccf1df' language='*'" -LIBPATH:C:\BW\WinCairo-EWS\build\WebKitBuild\Release\lib64   -LIBPATH:C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64 lib64\WebKit2.lib  lib64\WebCore.lib  lib64\JavaScriptCore.lib  ws2_32.lib  wsock32.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\libxml2.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\sqlite3.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\libxslt.lib  lib64\libEGL.lib  lib64\libGLESv2.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\lcms2.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\woff2dec.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\cairo.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\libcurl_imp.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\psl.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\ssl.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\crypto.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\jpeg.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\libpng16.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\zlib.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\openjp2.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\webpdemux.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\webp.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\jxl.lib  crypt32.lib  iphlpapi.lib  usp10.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\woff2common.lib  brotlidec.lib  lib64\WTF.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\CFLite.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\icudt.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\icuin.lib  C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\icuuc.lib  DbgHelp.lib  shlwapi.lib  winmm.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
> LINK: command "C:\MSVS\VC\Tools\MSVC\14.33.31629\bin\Hostx64\x64\link.exe /nologo Source\WebKit\CMakeFiles\WebProcess.dir\WebProcess\EntryPoint\win\WebProcessMain.cpp.obj /out:bin64\WebKitWebProcess.exe /implib:lib64\WebKitWebProcess.lib /pdb:bin64\WebKitWebProcess.pdb /version:0.0 /machine:x64 /DEBUG /OPT:ICF /OPT:REF /INCREMENTAL:NO /INCREMENTAL:NO /subsystem:console /manifestdependency:type='win32' name='Microsoft.Windows.Common-Controls' version='6.0.0.0' processorArchitecture='amd64' publicKeyToken='6595b64144ccf1df' language='*' -LIBPATH:C:\BW\WinCairo-EWS\build\WebKitBuild\Release\lib64 -LIBPATH:C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64 lib64\WebKit2.lib lib64\WebCore.lib lib64\JavaScriptCore.lib ws2_32.lib wsock32.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\libxml2.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\sqlite3.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\libxslt.lib lib64\libEGL.lib lib64\libGLESv2.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\lcms2.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\woff2dec.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\cairo.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\libcurl_imp.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\psl.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\ssl.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\crypto.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\jpeg.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\libpng16.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\zlib.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\openjp2.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\webpdemux.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\webp.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\jxl.lib crypt32.lib iphlpapi.lib usp10.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\woff2common.lib brotlidec.lib lib64\WTF.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\CFLite.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\icudt.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\icuin.lib C:\BW\WinCairo-EWS\build\WebKitLibraries\win\lib64\icuuc.lib DbgHelp.lib shlwapi.lib winmm.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:bin64\WebKitWebProcess.exe.manifest" failed (exit code 1120) with the following output:
> MSVCRT.lib(exe_main.obj) : error LNK2019: unresolved external symbol main referenced in function "int __cdecl __scrt_common_main_seh(void)" (?__scrt_common_main_seh@@YAHXZ)
> bin64\WebKitWebProcess.exe : fatal error LNK1120: 1 unresolved externals

This change disables precompiled header for WebKit module. Please don't do that.